src/block-components/progress-bar/style.js

```diff
@@ -30,6 +30,9 @@ const Styles = props => {
 				styleRule="--progress-value"
 				attrName="progressValue"
 				key="progressValue"
+				valueCallback={ value => {
+					return getDynamicContent( value ).replace( /,/g, '' )
+				} }
```

Comment:
Please add checks whether value is present first before you run it on `.replace`, this might produce an error if the value is not present or the value is invalid. `value` can also be undefined, try using `value || ''`